rest/route/aws.go

```diff
@@ -78,13 +94,7 @@ func (aws *awsSns) Run(ctx context.Context) gimlet.Responder {
 			"topic_arn":       aws.payload.TopicArn,
 		})
 	case messageTypeNotification:
-		// TODO: handle the message
-		grip.Info(message.Fields{
-			"message":         "got an AWS SNS notification",
-			"payload_subject": aws.payload.Subject,
-			"payload_message": aws.payload.Message,
-			"payload_topic":   aws.payload.TopicArn,
-		})
+		return aws.handleNotification(ctx)
```

Comment:
I think we should log any error from handleNotification, since AWS is the one calling the route but we may also want to know about the error

Reply:
done